recipes/glib-networking/post-link.bat

```diff
@@ -0,0 +1,3 @@
+"%PREFIX%\Library\bin\gio-querymodules.exe" "%PREFIX%\Library\lib\gio\modules"
```

Comment:
What is this doing?

Reply:
It creates a cache file with references to all installed Gio modules, which ends up getting used by Gio to discover them when necessary. Since these modules can be installed by other packages as well, the cache file should be regenerated in a post-install script. This is very similar to what happens for example in [gtk3](https://github.com/conda-forge/gtk3-feedstock/blob/master/recipe/post-link.sh).